charm/docs/explanation/charm-architecture.md

```diff
@@ -0,0 +1,31 @@
+# NetBox charm architecture
+
+The NetBox charm initial version has been generated with the 
+help of the paas-app-charmer project. The paas-app-charmer project provides many
```

Comment:
Maybe like to the [paas-app-charmer piwheels](https://www.piwheels.org/project/paas-app-charmer/) or the [github page](https://github.com/canonical/paas-app-charmer/)?

Reply:
Added link to the github page. Thanks!